irods_testing_environment/irods_setup.py

```diff
@@ -384,6 +406,8 @@ def restart_rsyslog(container):
         \$template irods_format,\\"%msg%\\n\\"
         :programname,startswith,\\"irodsServer\\" /var/log/irods/irods.log;irods_format
         & stop
+        :programname,startswith,\\"irodsAgent\\" /var/log/irods/irods.log;irods_format
+        & stop
```

Comment:
This only applies to iRODS 5, but is completely safe for iRODS 4. It's probably a good idea to only include this for iRODS 5. That way, people who use this as a reference don't get confused.

However, docs.irods.org documents the minimum requirements, so it's really a matter of where people should look for understanding the system: the testing env vs the docs

Thoughts?

Reply:
I agree.

Leaving as is.